src/main/resources/logback.xml

```diff
@@ -0,0 +1,52 @@
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="AllAppender" class="ch.qos.logback.core.FileAppender">
+        <file>log/all.log</file>
+        <append>false</append>
+        <immediateFlush>true</immediateFlush>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="ModelAppender" class="ch.qos.logback.core.FileAppender">
+        <file>log/model.log</file>
+        <append>true</append>
+        <immediateFlush>true</immediateFlush>
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="EvaluationAppender" class="ch.qos.logback.core.FileAppender">
+        <file>log/evaluator.log</file>
+        <append>true</append>
```

Comment:
why true?

Reply:
same as above, keeping the previous logs will help us to compare the different models we are using.

Reply:
redundant

Reply:
Done 👍